src/pages/_app.js

```diff
@@ -15,6 +16,9 @@ import { PageTitle } from 'components/shared/PageTitle'
 
 getPageLoader()
 const Fallback = () => <ErrorPage />
+const GA4MeasurementID =
+  process.env.GA4_MEASUREMENT_ID ||
+  process.env.NEXT_PUBLIC_DEV_GA4_MEASUREMENT_ID
```

Comment:
We don't need to do this. We should be setting the `STAGE_GA4_MEASUREMENT_ID` in `docker-compose.env`

Reply:
Based on the merged PR #296,  the `docker-compose.env` file is now excluded from commits. 

For local development, please note that the measurement ID used is the dev GA property `CCDL > Refine.bio Web - Dev` in our analytics account. Thank you David!